config/locales/university/en.yml

```diff
@@ -272,7 +272,7 @@ en:
         period: Period
       experiences:
         import_btn: Import experiences
-        import_hint_html: "Possible values for <i>gender</i> are: m (male), f (female) and n (non binary).<br><i>Phone_professional</i>, <i>phone_personal</i>, <i>mobile</i> and <i>zipcode</i> fields must have a text format, not numbers.<br><i>Country</i> field must contain the ISO 3166 code of the country, so 2 upcase characters (<a href=\"https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes\" target=\_blank\">list</a>).<br><i>Social_twitter</i> field should have no @."
+        import_hint_html: "<i>Company_siren</i> and <i>company_nic</i> fields must have a text format, not numbers.<br><i>experience_from</i> field should be the start year of the experience.<br><i>experience_to</i> field should be the end year of the experience. It can be blank is the experience is still actual."
```

Comment:
```suggestion
        import_hint_html: "<i>Company_siren</i> and <i>company_nic</i> fields must have a text format, not numbers.<br><i>experience_from</i> field should be the start year of the experience.<br><i>experience_to</i> field should be the end year of the experience. It can be blank if it's a current experience."
```